static/app/components/feedback/list/feedbackListItem.tsx

```diff
@@ -189,4 +189,10 @@ const BottomGrid = styled('div')`
   overflow: hidden;
 `;
 
+const StyledProjectAvatar = styled(ProjectAvatar)`
+  && img {
+    box-shadow: none;
+  }
+`;
```

Comment:
Would it make more sense to have a property that disables the box shadow I wonder?

Reply:
Or we could give the component space (margin), so the shadow isn't cutoff -> lay the blame on the component for not being easy to drop in place.
Or change the container here to it's not `overflow:hidden` (i'm assuming it is) -> lay the blame on the container here, for hiding content it shouldn't be.

idk, the avatars are a mess, i'd prefer to keep their props narrow.

Reply:
discussed offline, let's keep it the way it is to keep the scope local to our changes!